ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -29,6 +29,7 @@ import com.daml.ledger.api.refinements.{ApiTypes => lar}
 import com.daml.ledger.api.v1.{value => v}
 import com.daml.ledger.client.LedgerClient
 import com.daml.ledger.service.MetadataReader
+import com.daml.ledger.test.ModelTestDar
```

Comment:
Why did the name change?

Reply:
I introduced a sealed trait and case objects to more type-safe and less stringly-typed :)